dependabot/dependabot.yaml

```diff
@@ -70,7 +70,7 @@ files:
   - repo: openssh-keys
     path: .github/dependabot.yml
     vars:
-      dependabot_labels: []
+      dependabot_labels: [dependency, skip-notes]
```

Comment:
What is the intention with the `dependency` label ?

Reply:
It's just a way to search for dependency updates in the web UI.  No automation uses it.  openssh-keys didn't have it for some reason, but I've added it now.